.gitignore

```diff
@@ -0,0 +1,11 @@
+# Dependency directory
+# Commenting this out is preferred by some people, see
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+client/node_modules
+client/bower_components
+
+
+# Webstorm folder
+.idea
+
+client/**/*.css
```

Comment:
We've already got a `.gitignore` under `client` (currently just ignores `node_modules`). Instead of this top-level file, could you just add `bower_components` and `*.css` to it? You can also add `.idea` to your personal `.git/info/exclude` so it doesn't clutter the repo for people who don't use it. Then we can ditch this file.